manual/oscdoc_receivermod_hrtf.tex

```diff
@@ -18,6 +18,8 @@
 \attr{/.../freq\_end} & f &  & yes & \\
 \attr{/.../freq\_start} & f &  & yes & \\
 \attr{/.../gaincorr} & ff &  & no & channel-wise gain correction\\
+\attr{/.../nf\_filter} & i & & yes & \\
```

Comment:
It is possible to provide brief documentation as an additional parameter in the add_double() / add_float() etc methods of the OSC server. Below it seems to be provided correctly, however, the documentation is not shown here. Needs to be investigated.